types/database/localDatabase.types.ts

```diff
@@ -3,6 +3,10 @@ import {SQLiteDatabase} from 'react-native-sqlite-storage';
 interface UseLocalDatabaseHook {
   localDb: SQLiteDatabase;
   refresh: (key: 'channelList' | 'channelInfo' | 'chat' | 'user' | 'channelMember') => void;
+  refreshWithId: (
+    key: 'channelList' | 'channelInfo' | 'chat' | 'user' | 'channelMember',
+    id: string
+  ) => void;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `refreshWithId` function is a good improvement. It allows refreshing specific items in the database by their ID, which can be more efficient than refreshing all items of a certain type. However, it would be better to use an enum for the key parameter instead of hardcoding the strings. This will help prevent typos and make the code easier to maintain.
```diff
- key: 'channelList' | 'channelInfo' | 'chat' | 'user' | 'channelMember',
+ key: DatabaseKeys,
```
Where `DatabaseKeys` is an enum defined as:
```typescript
enum DatabaseKeys {
  ChannelList = 'channelList',
  ChannelInfo = 'channelInfo',
  Chat = 'chat',
  User = 'user',
  ChannelMember = 'channelMember'
}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->